travvy/src/components/Profile.vue

```diff
@@ -0,0 +1,7 @@
+<template>  
+  <div class="Profile">
+        <h2>Profile</h2>
+        <br>
+        <br>
```

Comment:
This is good. Maybe soon we can try to display the name to make sure the user's email from the store.